### PR TITLE
Added neo4j and topics-rw-neo4j

### PR DESCRIPTION
**Pull request by davidbalazs93**

### CODE REVIEW

File: kube-config-files/neo4j-red-service.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+metadata:
+  name: neo4j-red
+  labels:
+    app: neo4j-red
+    visualize: "true"
+spec:
+  ports:
+    - name: work
+      port: 7474
+  clusterIP: None
```
Comment (sbuliarca) [inline kube-config-files/neo4j-red-service.yaml]: I don't think this should be a headless service, as we want to use it to access the neo4j pod

File: kube-config-files/neo4j-red-service.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      app: neo4j-red
+  template:
+    metadata:
+      labels:
+        app: neo4j-red
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: neo4j-red
```
Comment (sbuliarca) [inline kube-config-files/neo4j-red-service.yaml:37]: Let's define a liveness probe for this also

**Review by sbuliarca — COMMENTED** [review]
This looks good overall, just some small details.

File: kube-config-files/neo4j-red-service.yaml (hunk 1 above)
Comment (davidbalazs93) [reply kube-config-files/neo4j-red-service.yaml]: as discussed, I will remove the clusterIP:none spec

File: kube-config-files/neo4j-red-service.yaml (hunk 2 above)
Comment (davidbalazs93) [reply kube-config-files/neo4j-red-service.yaml:37]: added liveness probe

**Review by sbuliarca — APPROVED** [review]

**Review by tamas-molnar — APPROVED** [review]

**Review by MMoisa — APPROVED** [review]

**Pull request merged**